quickwit/quickwit-query/src/query_ast/full_text_query.rs

```diff
@@ -316,6 +316,7 @@ mod tests {
     use crate::{create_default_quickwit_tokenizer_manager, BooleanOperand};
 
     #[test]
+    #[ignore]
```

Comment:
the behavior this test tries for changed, I don't think it make sense to keep it